testdata/networking/sriov/sriovnetworkpolicy/wrong/non-vondor

```diff
@@ -0,0 +1,17 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: without-netdevice
+  namespace: openshift-sriov-network-operator
+spec:
+  mtu: 1500
+  nicSelector:
+    pfNames:
+      - ens3f1
+    rootDevices:
+      - '0000:5e:00.1'
+    vendor: '15b4'
```

Comment:
File named `wrong/non-vondor`, but with vendor ?

Reply:
The correct vendor is '15b3', here is I'm using a wrong name `15b4` to check the validation.